### DISCUSSION TITLE
Allow filtering based on a user's online favorite beatmaps.

### DISCUSSION THREAD

**Discussion by dumbuz** [ppy/osu#22798] [Ideas] [answered]
In osu!stable, you can very easily filter to only show beatmaps that you have marked as your favorites. 
![image](https://user-images.githubusercontent.com/44739965/223033087-caa2834e-3dd0-4a4f-a248-70defe86a78d.png)

I can't find an option to do this in lazer, and you can seemingly only filter by local collections.
![image](https://user-images.githubusercontent.com/44739965/223033415-aa1328dc-747d-4c25-8006-b8ef12501eb6.png)

 This could work exactly how it did in osu!stable, only showing beatmaps that are downloaded, but since osu!lazer is so interconnected with osu!web, maybe it could show online metadata for beatmaps that aren't locally downloaded and have a button (similar to the Update button on non-up-to-date beatmaps) to download these charts.

**Comment by mk56-spn** [comment]
Very much second this idea sounds nice to have, have the feeling the second part was already suggested but just for any beatmap ( you would see placeholders for the maps in the carousel for any beatmap available on web )

**Reply by dumbuz** [reply]
> have the feeling the second part was already suggested but just for any beatmap ( you would see placeholders for the maps in the carousel for any beatmap available on web )

What? Is this for any beatmap on bancho, or only beatmaps you've previously played?

**Comment by tadatomix** [comment, marked as answer]
I think we'll finally get this with #34744
